staging/cert-manager-setup/templates/post-install-hook-job.yaml

```diff
@@ -20,4 +20,4 @@ spec:
       - name: {{ .Chart.Name }}
         image: bitnami/kubectl:1.18.8-debian-10-r15
         imagePullPolicy: IfNotPresent
-        command: ["kubectl", "wait", "--for=condition=Available", "--timeout=360s", "apiservice", "v1beta1.webhook.certmanager.k8s.io"]
+        command: ["sleep", "30"]
```

Comment:
I've tried `kubectl wait -n {{ .Release.Namespace }} --for=condition=Ready pods -l app=webhook`, which succeeds before the dynamic server certificate is populated in the service. This results in the post install operations  (issuer and  certificate creation) failing. This sleep duration seems to work consistently on my local machine and also for other cert-manager users in the wild. Still, I think it would be best to block until a test issuer creation succeeds. Here is a jira for this follow up: https://jira.d2iq.com/browse/D2IQ-71954

Reply:
If helm is run with `--wait` doesn't that wait for the deployment to be ready?

Reply:
It does, but the deployment becomes Available when all containers in the Pod template become `Ready`. Unfortunately, this happens before the service certificate is generated. So, using --wait, has the same issue unfortunately.